chef-server-stats/chef-server-stats

```diff
@@ -295,6 +316,7 @@ output.merge!(get_authz_stats(services))
 output.merge!(get_redis_stats(services))
 output.merge!(get_server_status)
 output.merge!(get_folsom_metrics(chef_server_12))
+output.merge!(get_pooler_metrics(chef_server_12))
```

Comment:
I was able to confirm, on a chef server version earlier then 12.2 we simply receive no additional metrics for Folsom or Pooler:

`[root@chef chef-server-stats]# ./chef-server-stats
WARNING: No knife configuration file found
{"chef.server.rabbitmq_messages_ready":0,"chef.server.postgresql_seq_scan":"742","chef.server.postgresql_seq_tup_read":"21216","chef.server.postgresql_idx_scan":"32474","chef.server.postgresql_idx_tup_fetch":"48102","chef.server.postgresql_n_tup_ins":"3424","chef.server.postgresql_n_tup_upd":"503","chef.server.postgresql_n_tup_del":"520","chef.server.postgresql_n_live_tup":"12086","chef.server.postgresql_n_dead_tup":"654","chef.server.postgresql_connection_count":21,"chef.server.status":1}
[root@chef chef-server-stats]# rpm -qa | grep chef
chef-12.15.19-1.el6.x86_64
chef-server-core-12.0.8-1.el6.x86_64`